### ISSUE TITLE
"managedsoftwareupdate --show-config" fails

### ISSUE THREAD

**Issue by workflowsguy** [munki/munki#1075]
`sudo managedsoftwareupdate -V`
->
`5.2.2.4286`

`sudo managedsoftwareupdate --show-config`
->
```
Current Munki configuration:
                   AdditionalHttpHeaders:  None [not set] 
        AggressiveUpdateNotificationDays:    14 [/Library/Preferences/ManagedInstalls.plist] 
Traceback (most recent call last):
  File "/usr/local/munki/managedsoftwareupdate", line 1218, in <module>
    main()
  File "/usr/local/munki/managedsoftwareupdate", line 689, in main
    prefs.print_config()
  File "/usr/local/munki/munkilib/prefs.py", line 310, in print_config
    value = false
NameError: name 'false' is not defined`
```

**Comment by aysiu** [comment]
Fixed in this PR: https://github.com/munki/munki/pull/1074

**Comment by gregneagle** [comment]
Fix was officially released with Munki 5.2.3.
